File: apps/home-service-platform-server/src/APIs/ServiceCategory/Dtos/ServiceCategoryCreateInput.cs
namespace HomeServicePlatform.APIs.Dtos;

public class ServiceCategoryCreateInput
{
    public List<Booking>? Bookings { get; set; }

    public string? CategoryName { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Description { get; set; }

    public string? Id { get; set; }

    public ServiceProvider? ServiceProvider { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: apps/home-service-platform-node-js-admin/src/api/serviceCategory/ServiceCategoryCreateInput.ts
import { BookingCreateNestedManyWithoutServiceCategoriesInput } from "./BookingCreateNestedManyWithoutServiceCategoriesInput";
import { ServiceProviderWhereUniqueInput } from "../serviceProvider/ServiceProviderWhereUniqueInput";

export type ServiceCategoryCreateInput = {
  bookings?: BookingCreateNestedManyWithoutServiceCategoriesInput;
  categoryName?: string | null;
  description?: string | null;
  serviceProvider?: ServiceProviderWhereUniqueInput | null;
};
